### PR TITLE
Fixed bug inside add photo form

### PR DESCRIPTION
**Pull request by eraulo**
- If user save the photo, user will get more information or note about to add more photo.
- User can skip when user don't have more photo  and also user can see the photo, user  have uploaded. 
- If user skip, it will going to home page.

### CODE REVIEW

File: map/templates/map/phototimor_form.html [hunk 1]
```diff
@@ -6,6 +6,11 @@
 {% block title %}{% trans 'Hatama photo istoria viazen' %}{% endblock %}
 
 {% block content %}
+<style>
+    .active{
+        color: black!important;
+    }
+</style>
```
Comment (marobo) [inline map/templates/map/phototimor_form.html:13]: Muda style nee ba site.css

File: map/templates/map/phototimor_form.html [hunk 2]
```diff
@@ -16,10 +21,22 @@
   <div class="modal-dialog modal-lg">
     <div class="modal-content">
         <div class="modal-header">
-            <h4 class="modal-title">{% trans 'Hatama photo istoria viazen' %}</h4>
+            {% if counts %}
+                <h4 class="modal-title">{% trans 'Favor bele hatama tan photo istoria viazen' %}</h4>
+            {% else %}
+                <h4 class="modal-title">{% trans 'Hatama photo istoria viazen' %}</h4>
+            {% endif %}
```
Comment (marobo) [inline map/templates/map/phototimor_form.html:28]: Texts hotu hotu tenki uza ingles

File: map/views.py [hunk 3]
```diff
@@ -49,6 +50,13 @@ class PhotoViazenView(CreateView):
     def get_success_url(self):
         return reverse_lazy('photo_viazen', args = (self.object.istoriaviazen_id,))
 
+    def get_context_data(self, *args, **kwargs):
+        target = self.kwargs['pk']
+        context = super(PhotoViazenView, self).get_context_data(*args, **kwargs)
+        context['photo'] = PhotoTimor.objects.filter(istoriaviazen=target)
```
Comment (marobo) [inline map/views.py:56]: context name tenki uza liafuan plural ex.  `foto_sira` ka `photos`

File: map/templates/map/phototimor_form.html [hunk 4]
```diff
@@ -6,6 +6,11 @@
 {% block title %}{% trans 'Hatama photo istoria viazen' %}{% endblock %}
```
Comment (joanico) [inline map/templates/map/phototimor_form.html:6]: Ita bele uza liafuan english iha trans tag

File: map/templates/map/phototimor_form.html [hunk 5]
```diff
@@ -16,59 +16,88 @@
...
+
+                                  {% if field.help_text %}
+                                    <small class="form-text text-muted">{{ field.help_text }}</small>
+                                  {% endif %}
                                 </div>
                               {% endfor %}
+                        </table>
+                        <div class="modal-footer">
+                        {% if journey_photos.count %}
+                            <div class="text-muted h6">{% trans 'Note: If don\'t have another photo, please skip' %}</div>
```
Comment (PeteCoward) [inline map/templates/map/phototimor_form.html:77]:
```suggestion
                            <div class="text-muted h6">{% trans "Note: If don't have another photo, please skip" %}</div>
```

**Pull request merged**